plugin.json

```diff
@@ -24,7 +24,7 @@
                 "key": "JitsiURL",
                 "display_name": "Jitsi Server URL",
                 "type": "text",
-                "help_text": "The URL for your Jitsi server. For example, https://jitsi.example.com.",
+                "help_text": "The URL for your Jitsi server, for example https://jitsi.example.com. Defaults to https://meet.jit.si, which is the public server provided by Jitsi.",
                 "placeholder": "https://jitsi.example.com"
```

Comment:
Let's replace the placeholder with a default setting: https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema.settings.default

Reply:
Done!